libmscore/cmd.cpp

```diff
@@ -2391,31 +2398,45 @@ Element* Score::move(const QString& cmd)
 
             }
       else if (cmd == "next-measure") {
-            el = nextMeasure(cr);
-            if (noteEntryMode())
+            if (box && box->nextMeasure() && box->nextMeasure()->first())
+                  el = box->nextMeasure()->first()->nextChordRest(0, false);
+            if (cr)
+                  el = nextMeasure(cr);
+            if (el && noteEntryMode())
                   _is.moveInputPos(el);
             }
       else if (cmd == "prev-measure") {
-            el = prevMeasure(cr);
-            if (noteEntryMode())
+            if (box && box->prevMeasure() && box->prevMeasure()->first())
+                  el = box->prevMeasure()->first()->nextChordRest(0, false);
+            if (cr)
+                  el = prevMeasure(cr);
+            if (el && noteEntryMode())
                   _is.moveInputPos(el);
             }
-      else if (cmd == "next-system") {
+      else if (cmd == "next-system" && cr) {
```

Comment:
what about "next-system" when `cr` is falsy?

Reply:
No need

Reply:
what did it do before this PR?

Reply:
Although..., allowing _prev-system_ to execute when a valid frame is selected would add a little functionality for when going backwards. Otherwise, _next-measure_ would do what would be imagined _next-system_ would do when a valid frame is selected.

Reply:
Nothing. A valid ChordRest* was and is required.